testgrid/web/package.json

```diff
@@ -69,7 +68,7 @@
     "react-helmet": "^6.1.0",
     "react-modal": "^3.14.4",
     "react-monaco-editor": "^0.44.0",
-    "react-router-dom": "^6.0.2",
+    "react-router-dom": "^5.2.0",
```

Comment:
I went back because rolling forward breaks the properties the `Run` component is expecting, and the only fix option provided was using hooks.